src/Kubernetes.Controller/Protocol/Receiver.cs

```diff
@@ -53,7 +67,9 @@ public override async Task RunAsync(CancellationToken cancellationToken)
 
             try
             {
-                using var stream = await client.GetStreamAsync(_options.ControllerUrl, cancellationToken).ConfigureAwait(false);
+                var requestMessage = new HttpRequestMessage(HttpMethod.Get, _options.ControllerUrl);
+                var responseMessage = await _options.Client.SendAsync(requestMessage, HttpCompletionOption.ResponseHeadersRead, cancellationToken).ConfigureAwait(false);
```

Comment:
```suggestion
                var responseMessage = await _options.Client.SendAsync(requestMessage, cancellationToken).ConfigureAwait(false);
```

Mb, missed that this wasn't HttpClient anymore